horse-trainer.lic

```diff
@@ -19,11 +19,14 @@ class TeachHorse
   end
 
   def instruct(skill)
+    @start = Time.now
```

Comment:
These don't need to be instance vars if they're all in the same scope.

Reply:
thanks for the heads up!